tests/test_auth_enumeration.py

```
@@ -0,0 +1,160 @@
+        reseller_ids=list(resellers),
+        merchant_ids=list(merchants),
+        permissions=[],
+        owner_id=owner_id,
+    )
+
+
+# ── PT-16: account listing requires proof of ownership ────────────────────
+def test_list_accounts_request_requires_password_with_email():
+    from app.schemas.breeze_buddy.signup import ListAccountsRequest
+
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)